### PR TITLE
Include coverage in spec matrix

### PR DESCRIPTION
**Pull request by marcandre**
🤞

### CODE REVIEW

File: .github/workflows/rubocop.yml [hunk 1]
```diff
@@ -49,10 +52,20 @@ jobs:
           git clone https://github.com/rubocop-hq/rubocop.git ../rubocop
           chmod +x ../rubocop/exe/rubocop
           cd ../rubocop && bundle install --jobs 3 --retry 3
+      - name: code coverage
+        if: matrix.coverage == 'yes'
+        uses: paambaati/codeclimate-action@v2.6.0
+        env:
+          CC_TEST_REPORTER_ID: '758a8228862932dc8afa9144c4a5bc5dfe29c2f7dde1b7734175bad49ee310e7'
```
Comment (bbatsov) [inline .github/workflows/rubocop.yml:59]: Can't this be configured somewhere in GitHub Actions's UI? It's not a sensitive token, but I think it's best to keep tokens outside the source.

Comment (marcandre) [reply .github/workflows/rubocop.yml:59]: There is support for secrets somewhere. Is it ok to set it up like this for now and I'll leave an issue?
I find the doc of the github actions really difficult to search through.

Comment (bbatsov) [reply .github/workflows/rubocop.yml:59]: Yea, that's fine.

**Comment by marcandre** [comment]
I hacked the title for the steps 💪, coverage report seem to be received on Code Climate, but the badge / dashboard page don't show the coverage.
I'll merge this for now, maybe it's just because it's not in master yet?

**Comment by marcandre** [comment]
Yay, it worked 🎉

**Pull request merged**